### ISSUE TITLE
prop-types false negative in stateless component outside of jsx

### ISSUE THREAD

**Issue by billyjanitsch** [yannickcr/eslint-plugin-react#486]
``` js
export const MyComponent = props => props.test
  ? <div />
  : <span />
```

Expected: `props.test` should throw a `prop-types` error because MyComponent has no propTypes validation.

eslint: 2.2.0
eslint-plugin-react: 4.1.0
